securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -5,9 +5,9 @@
 
 The library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive.
 
-A best practice is to load the JavaScript adapter directly from {project_name} Server as it will automatically be updated when you upgrade the server. If you copy the adapter to your web application instead, make sure you upgrade the adapter only after you have upgraded the server.
+A good practice is to include the JavaScript adapter directly into your web application to make sure your web application is self-contained and starts up fast. Keep in mind you need to upgrade the JavaScript adapter after you have upgraded the server. Alternatively, load the JavaScript adapter directly from {project_name} Server. Then it will automatically be updated when you upgrade the server.
```

Comment:
Copying the adapter code into your own application should never be a recommendation in my opinion. Folks should use a system that allows them to manage dependencies such as NPM so they can get timely updates and security warnings.

Reply:
thx for your input! Makes sense for me. Then i'll update the text here and also create a PR to update https://www.keycloak.org/server/reverseproxy#_exposed_path_recommendations to state "A good practice is to not use external js for the javascript client, but add it into your public client instead. The keycloak js package is available at https://www.npmjs.com/package/keycloak-js" wdyt?

Reply:
Yes, that sounds like a plan :+1:

Reply:
@jonkoops I've put out an updated version, also linking the yarn package (didn't want to advertize one or the other, so added both ;) ) could you re-check?